--- Makefile.orig	2026-04-19 15:12:05
+++ Makefile	2026-04-19 15:12:17
@@ -70,6 +70,11 @@
   LDFLAGS := -Wl,-dead_strip $(MINVERSION)
   COMMONFLAGS += -DDISABLE_DYNAREC=1 -Wno-unknown-warning-option -Wno-deprecated-declarations $(MINVERSION)
   STRIPCMD := $(or $(STRIP),strip) -xS
+else ifeq ($(platform), emscripten)
+  OUTNAME := dosbox_pure_libretro_$(platform).bc
+  STATIC_LINKING = 1
+  CXX += -std=gnu++11
+  COMMONFLAGS += -DEMSCRIPTEN -pthread
 else ifneq ($(ISMAC),)
   OUTNAME := dosbox_pure_libretro.dylib
   CXX     ?= c++
